HW28/Dockerfile

```diff
@@ -0,0 +1,17 @@
+
+FROM python:3.9
+
+
+WORKDIR /app
+
+
+COPY . /app
+
+
+RUN pip install --no-cache-dir -r requirements.txt
```

Comment:
It will not work for someone else, because there is no `requirements.txt` in the repository